store/metadata/bin/boot

```diff
@@ -47,6 +47,11 @@ if ! etcdctl --no-sync -C $ETCD get ${ETCD_PATH}/filesystemSetupComplete >/dev/n
   fi
 fi
 
+until confd -onetime -node $ETCD --confdir /app --log-level error; do
```

Comment:
isn't this covered by lines 11-14? If that passes, the keyring file should be templated. Odd...